src/xdocs/config.xml

```diff
@@ -242,9 +242,7 @@
 
       <source>
 &lt;module name=&quot;JavadocMethod&quot;&gt;
-  &lt;property name=&quot;severity&quot;
-               value=&quot;${checkstyle.javadoc.severity}&quot;
-               default=&quot;error&quot;/&gt;
+  &lt;property name=&quot;severity&quot; value=&quot;${checkstyle.javadoc.severity}&quot; default=&quot;error&quot;/&gt;
```

Comment:
Line must remain under 100 characters.
We don't have this rule integrated by CI yet.
Please ensure all lines you change remain under 100.

Reply:
done